processing/src/main/java/io/druid/query/filter/DimFilterCacheHelper.java

```diff
@@ -37,7 +37,7 @@
   static final byte JAVASCRIPT_CACHE_ID = 0x7;
   static final byte SPATIAL_CACHE_ID = 0x8;
   static final byte IN_CACHE_ID = 0x9;
-  static final byte STRING_SEPARATOR = (byte) 0xFF;
+  public static final byte STRING_SEPARATOR = (byte) 0xFF;
```

Comment:
if I am implementing a new type of filter, how should I declare the ID? Why is IN_CACHE_ID different from BOUND_CACHE_ID?